### ISSUE TITLE
The system is blocked by missing device.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [OpenZWave/open-zwave#162]
```
Step to reproduce it
1. Add two devices.
2. Remove the power of one of them.
3. The system is stalled to wait for the device with the power. --> problem


```

Original issue reported on code.google.com by `wyc...@gmail.com` on 23 Jan 2013 at 8:09
- Merged into: #164

Attachments:
- [BlockByAMissingDevice.txt](https://storage.googleapis.com/google-code-attachments/open-zwave/issue-162/comment-0/BlockByAMissingDevice.txt)
